blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineRunImpl.java

```diff
@@ -1,5 +1,8 @@
 package io.jenkins.blueocean.rest.impl.pipeline;
 
+import com.github.benmanes.caffeine.cache.Cache;
+import com.github.benmanes.caffeine.cache.CacheLoader;
+import com.github.benmanes.caffeine.cache.Caffeine;
```

Comment:
Being loaded from the plugin wrapper?

Reply:
yup. sadly by dependency transitivity and not directly but I don't really want touch too much poms in a maintenance branch.